CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Keycloak Community
+
+The Keycloak Node.js adapter is an Open Source adapter solution to protect modern Node.js Applications and Services.
+
+## Contributing to the Node.js adapter
```

Comment:
Why do we reference the Node,js adapter here ?

Reply:
Thanks updated.